docs/standard/garbage-collection/large-object-heap.md

```diff
@@ -0,0 +1,313 @@
+---
+title: "The large object heap"
+ms.custom: ""
+ms.date: "05/02/2018"
+ms.prod: ".net"
+ms.technology: dotnet-standard
+ms.topic: "article"
+helpviewer_keywords: 
+  - large object heap (LOH)"
+  - LOH
+  - "garbage collection, large object heap"
+  - "GC [.NET ], large object heap"
+author: "rpetrusha"
+ms.author: "ronpet"
+ms.workload: 
+  - "dotnet"
+  - "dotnetcore"
+---
+# The large object heap
+
+The .NET Garbage Collector (GC) divides objects up into small and large objects. When an object is large, some of its attributes become more significant than if the object is small. For instance, compacting it -- that is, copying it in memory elsewhere on the heap -- can be expensive. Because of this, the .NET Garbage Collector places large objects on the large object heap (LOH). In this topic, we'll look at the large object heap in depth. We'll discuss what qualifies an object as a large object, how these large objects are collected, and what kind of performance implications large objects impose.
+
+## How an object ends up on the large object heap and how GC handles them.
+
+If an object greater than or equal to 85,000 bytes, it’s considered a large object. This number was determined by performance tuning. When an object allocation request is 85,000 or more bytes, the runtime allocates it on the large object heap.
+
+To understand what this means, it's useful to examine some fundamentals about the .NET GC.
+
+The .NET Garbage Collector is a generational collector. It has three generations: generation 0, generation 1, and generation 2. The reason for for having 3 generations is, in a well-tuned app, most objects die in Gen0. For example, in a server app, the allocations associated with each request should die after the request is finished. The in-flight allocation requests will make it into Gen1 and die there. Essentially, Gen1 acts as a buffer between young object areas and long-lived object areas.
+
+Large objects belong to generation 2 because they are collected only during a generation 2 collection. When a generation is collected, all its younger generation(s) are also collected. For example, when a generation 1 GC happens, both generation 1 and 0 are collected. And when a generation 2 GC happens, the whole heap is collected. For this reason, a generation 2 GC is also called a *full GC*. This article refers to generation 2 GC instead of full GC, but the terms are interchangeable.
```

Comment:
These two paragraphs imply but don't state a couple points:
1. that small objects can survive generation 1 and enter generation 2.
1. that large objects are part of generation 2 *as soon as they are allocated.* 

It might be worth stating those explicitly (if the statements are correct) /cc @Maoni0 

Update: Both these statements are made clear below. It's your choice if you want to surface them here @rpetrusha